app/utils/request_logger.js

```diff
@@ -1,34 +1,53 @@
 const logger = require('./logger')(__filename)
+const { keys } = require('@govuk-pay/pay-js-commons').logging
 
 module.exports = {
   logRequestStart: context => {
-    logger.debug(`Calling ${context.service}  ${context.description}-`, {
+    const logContext = {
       service: context.service,
       method: context.method,
-      url: context.url
-    })
+      url: context.url,
+      description: context.description
+    }
+    logContext[keys.CORRELATION_ID] = context.correlationId
+    logger.info(`Calling ${context.service} to ${context.description}`, logContext)
   },
 
-  logRequestEnd: context => {
+  logRequestEnd: (context, response) => {
     let duration = new Date() - context.startTime
-    logger.info(`[${context.correlationId}] - ${context.method} to ${context.url} ended - elapsed time: ${duration} ms`)
+    const logContext = {
+      service: context.service,
+      method: context.method,
+      url: context.url,
+      description: context.description,
+      response_time: duration,
+      status: response && response.statusCode
```

Comment:
It's a bit confusing that until you added `status` here this would have been guaranteed logged exactly the same thing for both failures and successes (so a positive change!). 

We call `logRequestEnd` no matter if it's a success, "failure" or "error". I wonder if would be easier to follow if `logRequestFailure` and `logRequestEnd` just did their own `duration` calculation and each state only had one self contained log line. 

In that case `logRequestEnd` would be more specifically `logRequestSucceeded`.

Reply:
I see what you mean, but also i think it's more useful to have a single line that we can see the status code from.

Maybe that would mean just getting rid of `logRequestFailure`?